src/lender/admin/pool.sol

```diff
@@ -11,31 +9,59 @@ interface LendingAdapterLike {
     function raise(uint256 amount) external;
     function sink(uint256 amount) external;
     function heal() external;
+    function file(bytes32 what, uint value) external;
+}
+
+interface FeedLike {
+    function overrideWriteOff(uint loan, uint writeOffGroupIndex_) external;
+    function file(bytes32 name, uint risk_, uint thresholdRatio_, uint ceilingRatio_, uint rate_, uint recoveryRatePD_) external;
+    function file(bytes32 name, uint rate_, uint writeOffPercentage_, uint overdueDays_) external;
+    function file(bytes32 name, uint value) external;
+    function file(bytes32 name, bytes32 nftID_, uint maturityDate_) external;
+    function update(bytes32 nftID_,  uint value) external;
+    function update(bytes32 nftID_, uint value, uint risk_) external;
 }
 
 interface MemberlistLike {
     function updateMember(address usr, uint256 validUntil) external;
     function updateMembers(address[] calldata users, uint256 validUntil) external;
 }
 
+interface CoordinatorLike {
+    function file(bytes32 name, uint value) external;
+    function file(bytes32 name, bool value) external;
+    function poolClosing() external view returns(bool);
+}
+
 // Wrapper contract for various pool management tasks.
-contract PoolAdmin is Auth {
+contract PoolAdmin {
   
-    AssessorLike public assessor;
-    LendingAdapterLike public lending;
-    MemberlistLike public seniorMemberlist;
-    MemberlistLike public juniorMemberlist;
+    AssessorLike        public assessor;
+    LendingAdapterLike  public lending;
+    FeedLike            public navFeed;
+    MemberlistLike      public seniorMemberlist;
+    MemberlistLike      public juniorMemberlist;
+    CoordinatorLike     public coordinator;
 
-    bool public live = true;
+    bool                public live = true;
 
-    // Admins can manage pools, but have to be added and can be removed by any ward on the PoolAdmin contract
-    mapping(address => uint256) public admins;
+    mapping (address => uint256) public level1_admins;
+    mapping (address => uint256) public level2_admins;
+    mapping (address => uint256) public level3_admins;
 
-    // Events
-    event Depend(bytes32 indexed contractname, address addr);
-    event File(bytes32 indexed what, bool indexed data);
-    event RelyAdmin(address indexed usr);
-    event DenyAdmin(address indexed usr);
+    modifier level1     { require(level1_admins[msg.sender] == 1 && live); _; }
+    modifier level2     { require(level2_admins[msg.sender] == 1 && live); _; }
+    modifier level3     { require(level3_admins[msg.sender] == 1 && live); _; }
```

Comment:
Currently a level3 admin could not call a level1 method. (If the same address is not added again for level1).

Is that the intention? If yes, maybe the `level` naming is a bit missleading.

However, we could also have just one mapping called `admin_level` with values from 1 to 3.  

The modifier would check for `<=`

Reply:
```javascript
// level 1
function name(...) public admin_level(1) 

// level 2
function name(...) public admin_level(2) 
```

Reply:
We could also keep the existing three different modifier but just with one mapping